src/tests/base_test.py

```diff
@@ -821,6 +822,16 @@ def start_ipxe_server(**kwargs):
             for server in ipxe_server_controllers:
                 server.remove()
 
+    @pytest.fixture()
+    def iscsi_target_controller(self) -> Callable:
+        log.info("--- SETUP --- iscsi server controller")
+        iscsi_server_controller = IscsiTargetController()
+
+        yield iscsi_server_controller
```

Comment:
```suggestion
        yield iscsi_target_controller
```

Reply:
Yes i return IscsiTargetController  and not callable ,  the TargetController is director for creating
@pytest.fixture()
    def iscsi_target_controller(self) -> IscsiTargetController:
        log.info("--- SETUP --- iscsi server controller")
        _iscsi_target_controller = IscsiTargetController()

        yield _iscsi_target_controller
        if global_variables.test_teardown:
            log.info("--- TEARDOWN --- iscsi controller")
            _iscsi_target_controller.clean_target()